tools/spotify-importer.js

```diff
@@ -53,47 +53,51 @@ import {resetStorageForTesting} from '../runtime/ts-build/storage/firebase-stora
         continue;
       }
 
+      let description = playlist.description.replace(/ (Cover:|Related:|Related lists:|Also see:).*/, '');
+      description = description.replace(/<[^>]+>/g, '');
+
       let thumbnail;
       if (playlist.images && playlist.images.length > 0) {
         thumbnail = playlist.images[0].url;
       }
 
-      let artists = [];
+      let artists = new Set();
       if (playlist.tracks && playlist.tracks.items) {
         for (let item of playlist.tracks.items) {
           if (item.track) {
             for (let artist of item.track.artists) {
               if (artist.name) {
-                artists.push(artist.name);
+                artists.add(artist.name);
               }
             }
           }
         }
       }
 
-      console.log(`${playlist.name}: ${artists.length} artists`);
+      console.log(`${playlist.name}: ${artists.size} artists`);
       await collection.store({
         id: `${idBase}:${index}`,
         name: playlist.name,
-        description: playlist.description,
+        description,
         thumbnail,
         link: playlist.external_urls && playlist.external_urls.spotify,
-        artists: artists.join('|')
+        artists: [...artists].join('|')
```

Comment:
what's the difference?

Reply:
artists is a Set now, and Sets don't support join.